docs/wiki/The CI environment - Troubleshooting.md

```diff
@@ -33,3 +33,21 @@ Potential reasons
     ```
 
   - **Background:** If configured as a multiline object, GitHub interprets each line as a dedicated secret value. This leads to a situation where characters like `{` & `}` are treated as individual secrets - and values that are passed as JSON strings between pipeline jobs that contain these characters cannot be passed anymore (as per GitHubs default behavior). For example: `{"removeDeployment": true}`. Because this value isn't passed anymore, subsequently the line `removeDeployment: '${{ (fromJson(needs.job_initialize_pipeline.outputs.workflowInput)).removeDeployment }}'` that references it throws the error message you're seeing.
+
+# Outdated Bicep CLI version
+
+> Related error messages:
+>
+> - `Error BCP018: Expected the ":" character at this location.`
+> - `Error BCP022: Expected a property name at this location.`
+> - `Error BCP236: Expected a new line or comma character at this location.`
+> - `...` with the described reason below, there can be any amount of error messages you may encounter
+
+Potential reasons
+
+- The Bicep CLI version is outdated.
+  - **Solution:** Update the version of Bicep you're using on your machine. This can include:
+    - The Bicep extension for VS Code (update using, for example, the VSCode UI)
+    - The Bicep extension for the Azure CLI (update executing, for example, `az bicep upgrade`)
+    - The Bicep CLI (update executing, for example, `choco bicep upgrade`)
```

Comment:
This won't work for those not using chocolatey, plus it's only relevant for Windows, let's copy the PS snippet or better reference the official docs page https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/install#azure-powershell

Reply:
Hence the "for example". Figured it would be better than nothing and gets the idea across

Reply:
Can update it tomorrow

Reply:
Updated and added a disclaimer to the front page as per some feedback I received